src/libstore/store-api.hh

```diff
@@ -158,7 +158,9 @@ public:
 
     const Setting<int> pathInfoCacheSize{this, 65536, "path-info-cache-size", "size of the in-memory store path information cache"};
 
-    const Setting<bool> isTrusted{this, false, "trusted", "whether paths from this store can be used as substitutes even when they lack trusted signatures"};
+    const Setting<bool> isTrusted{this, false, "trusted", "whether paths from this store can be used as substitutes even when they lack trusted signatures. The dual of \"trusting\""};
+
+    Setting<bool> isTrusting{this, true, "trusting", "whether (we think) paths an be be added to this store even when they lack trusted signatures. The dual of \"trusted\""};
```

Comment:
```suggestion
    Setting<bool> isTrusting{this, true, "trusting", "whether (we think) paths can be added to this store even when they lack trusted signatures. The dual of \"trusted\""};
```

Reply:
I think we need some example of when you want to set this true or false. Could we just ask the builder whether they trust us, eliminating the need for a setting?